### ISSUE TITLE
Wallpaper seen through area between upper rounded corners and wingpanel

### ISSUE THREAD

**Issue by Tidrek** [elementary/stylesheet#210]
When a desktop wallpaper is so bright that Wingpanel becomes transparent, it becomes black when a window is maximized. So are those tiny areas between it and two upper rounded corners of that window. But not with Terminal. It doesn't happen with dark wallpapers but with bright ones those tiny areas are not filled black, I can see my wallpaper there. Here how it looks like:

![Left corner][1] ![Right corner][2]

Maybe it's hard to see that on the pictures but you can try it yourself.
A small issue but drives me nuts every damn time!

**To recap, it happens only with terminal window maximized and a bright wallpaper set.**

[1]: https://i.stack.imgur.com/OWPxb.png
  [2]: https://i.stack.imgur.com/OBzOY.png

**Comment by danrabbit** [comment]
This is actually an issue in Terminal itself. Fixed in this PR: https://github.com/elementary/terminal/pull/152
